lms/templates/problem.html

```diff
@@ -5,38 +5,106 @@
 %>
 
 <%namespace name='static' file='static_content.html'/>
-<h3 class="hd hd-2 problem-header">
+<h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="${ id }-problem-progress" tabindex="-1">
```

Comment:
@cahrens `id` is the location of any block i.e. `block-v1:edX+DemoX+Demo_Course+type@problem+block@02ba1809d8584577aa703d7714e6a588-problem-title`  while `short_id` is `02ba1809d8584577aa703d7714e6a588` from this example.